modules/core/src/v2/coins/trx.ts

```diff
@@ -192,4 +205,47 @@ export class Trx extends BaseCoin {
 
     return Buffer.from(sig, 'hex');
   }
+
+  /**
+   * Explain a Tron transaction from txHex
+   * @param params
+   * @param callback
+   */
+  explainTransaction(
+    params: ExplainTransactionOptions,
+    callback?: NodeCallback<TransactionExplanation>
+  ): Bluebird<TransactionExplanation> {
+    return co<TransactionExplanation>(function*() {
+      const txHex = params.txHex || (params.transaction && params.transaction.txHex);
+      if (!txHex || !Tron.safeTronHexString(txHex)) {
```

Comment:
Seems a bit silly for this to be in tron lib (I looked at the implementation of this function). This could be replaced by `/^(0x)?([0-9a-f]{2})+$/i.match(txHex)`. Will leave a review on that PR as well.

Reply:
Plus how is that any different than a generic safe hex string? Could use a generic hex sanitizer

Reply:
It is indeed a generic hex checker. I took it from Platform since we don't have one in SDK AFAIK, but I agree that naming wis misleading. 

What about we take this out of the Tron lib and put it into SDK?

Reply:
Since it is a generic account-lib it could be just accountlib.utils.safeHexString(). Though, I personally prefer the `isValidHexString()` style which returns boolean valid/invalid, rather than the `safeHexString()` style which just returns undefined if unsafe